bricks/template/brick.yaml

```
@@ -46,3 +46,13 @@ vars:
     description: Your app version
     default: "1"
     prompt: Build number?
+  json_format:
+    type: enum
+    description: Your default Json formatting when making the network request
+    default: snake
+    prompt: Your default Json formatting?
```

- To make it match other fields, should we remove `Your` from the prompt & description?
- The correct name of this feature should be `Json field renaming`, right? 
  - https://www.youtube.com/watch?v=Y3ogKymvol0

```suggestion
    description: Default Json field renaming with json_serializable (https://pub.dev/packages/json_serializable#build-configuration)
    default: snake
    prompt: Default Json field renaming format?
```

---

@luongvo Fixed in 54af0f2